### ISSUE TITLE
Allow aria-setsize, aria-posinset on grids

### ISSUE THREAD

**Issue by shleewhite** [w3c/html-aria#226]
I have a use case where we have a complex data grid and for performance reasons, not all the rows are in the DOM at all times. The user scrolls and more rows are loaded in.

Looking at the current documentation, it says aria-setsize and aria-posinset are only allowed on treegrids. I think this should be expanded to elements with role="grid" as well.

**Comment by scottaohara** [comment]
This would be a request for the ARIA specification as this is asking for a change to accepted attributes for use with ARIA roles.  ARIA in HTML is specifically for indicating what ARIA is allowed on HTML elements.
